game/weather.py

```diff
@@ -168,12 +170,13 @@ def generate(
         time_of_day: TimeOfDay,
         settings: Settings,
     ) -> Conditions:
+        _start_time=cls.generate_start_time(theater, day, time_of_day, settings.night_disabled)
```

Comment:
I moved this to a local variable to prepare for a more sophisticated time-of-day based temperature generation, for a future PR.